test/test-network/systemd-networkd-tests.py

```diff
@@ -3583,6 +3583,7 @@ def test_sysctl(self):
         self.check_ipv6_sysctl_attr('dummy98', 'proxy_ndp', '1')
         self.check_ipv4_sysctl_attr('dummy98', 'forwarding', '1')
         self.check_ipv4_sysctl_attr('dummy98', 'proxy_arp', '1')
+        # FIXME: self.check_ipv4_sysctl_attr('dummy98', 'proxy_arp_pvlan', '1') should it end here?
```

Comment:
Please enable the test. Or doesn't it work?

Note, you need to update 25-sysctl.network.

Reply:
Well, there's a comment on this test saying it's unstable so...

Reply:
Done either case.